e2e/compare/compare.js

```diff
@@ -0,0 +1,146 @@
+/* eslint-disable @lwc/lwc/no-async-await,rulesdir/no-negated-variables */
+const fs = require('fs/promises');
+const fsSync = require('fs');
+const _ = require('underscore');
+const {OUTPUT_DIR} = require('../config');
+const {computeProbability, computeZ} = require('./math');
+const printToConsole = require('./output/console');
+const writeToMarkdown = require('./output/markdown');
+
+/*
+ * base implementation from: https://github.com/callstack/reassure/blob/main/packages/reassure-compare/src/compare.ts
```

Comment:
Would it might be easier to import this via `npm`?

Reply:
Yeah, I'd have loved to just use the npm package! However, their code is very specific to reassure. We'd need to make changes to their packages and make their code more open to allowing us to use it as a package.

However, I felt the work to make reassure's code more open wasn't worth it yet (would take some significant time and I prioritized getting this test suite ready).